### PR TITLE
Add migration script

### PR DESCRIPTION
**Pull request by pjanik**
[#172225524]

The TravisCI build is failing due to missing ruby binaries on travis. But I don't think it matters at this point.

I already ran this script using AWS Lightsail server, it took a few minutes to migrate 35k documents that we had on production. There were no errors during migration. So far I migrated to staging environment, as only this version of token-service supports read-write tokens.

I run the script using:
`AWS_ACCESS_KEY_ID=123 AWS_SECRET_ACCESS_KEY=xyz DATABASE_URL=postgres://abc npm start`
Before I had to copy there my Firebase key.json file. Comments should explain all that.

@scytacki, re handling documents with missing read key - finally I'm using a one-way hash to generate new document ID if the readWriteKey is available. CFM can decide whether it wants to support this case or just leave these docs as read-only. It doesn't seem to hurt and if we ever need to change the approach, we won't have to migrate docs again. If there's no readWriteKey, we can't do anything and I'm just generating a random ID.

### CODE REVIEW

File: token-service-migration/stats.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          if (biggestDocs.length > docsCount) {
+            biggestDocs.length = docsCount;
+          }
+        }
+      });
+
+      if (rows.length > 0) {
+        read();
+      } else {
+        log(`\nThe biggest documents: ${biggestDocs.map(v => (v/1e6).toFixed(3) + "MB").toString()}\n`);
```
Comment (scytacki) [inline token-service-migration/stats.js:76]: Not important to change, but instead of `1e6` it should be `Math.pow(2, 20)` or `2**20`

**Comment by pjanik** [comment]
@scytacki, this is ready for review again.

**Comment by pjanik** [comment]
Results of migration to token-service-staging. They might be important as some data has been updated in DocStore DB. All the IDs are reported here:
https://gist.github.com/pjanik/1a2e45c45c95ef28c99f6954e5ba9cae

**Comment by pjanik** [comment]
@scytacki, timestamp comparison is added. I tested it and it works as expected.
Also, there was pretty critical error in the INNER JOIN statement - I was trying to join `documents` and `document_contents` using `id` instead of `document_id` and `id`. So, all the migrated documents possibly had the wrong content actually. 😬I noticed that when I was testing timestamp feature manually.

Re timestamp, I tested that by creating a new shared document in CODAP. I noted its ID, and added `WHERE id = <my_id>` to SQL statements. Then, I ran migration script a few times, in the meantime updating document in CODAP. I was checking updated migration statistics to see if doc was skipped or uploaded again. The main issue was that pg-node is assuming that DB timestamp has no timezone (it's UTC actually) and then it uses local timezone, which is ofc breaking the logic. So, I've added TZ=UTC to package.json script. So, it's important to run the migration using npm script or ensure that timezone is set to UTC.

**Pull request merged**